### ISSUE TITLE
Release a new version with Python 3.7 support

### ISSUE THREAD

**Issue by benruijl** [dgrunwald/rust-cpython#152]
At the moment, the only way to compile `cpython` with Python 3.7 is to bypass crates.io:

```
[dependencies.cpython]
git = "https://github.com/dgrunwald/rust-cpython"
version = "0.2"
```

**Comment by benruijl** [comment]
Please update the crate on crates.io to version 0.2.1 with support for Python 3.7. I cannot publish my crate otherwise.

**Comment by dgrunwald** [comment]
I just released version 0.2.1.
